src/cloud_provider/helm.rs

```diff
@@ -108,6 +105,17 @@ impl ChartInfo {
         }
     }
 
+    pub fn new_from_release_name(name: &str, custom_namespace: &str) -> ChartInfo {
+        ChartInfo {
+            name: name.to_string(),
+            namespace: HelmChartNamespaces::Custom,
+            custom_namespace: Some(custom_namespace.to_string()),
+            timeout_in_seconds: 3600,
```

Comment:
Why did you changed the default timeout? 3600 is far too high

Reply:
There was no default timeout before, I added this function for creating chartInfo from old HelmChart which wasn't containing any timeout.
After that it was using default max value for it, so no timeout ;'

Reply:
..Default you're calling has a default one and 3600 is far too high